proxy/ha_proxy.go

```diff
@@ -227,6 +229,14 @@ func (m HaProxy) getConfigData() ConfigData {
     option  dontlognull
     option  dontlog-normal`
 	}
+
+	default_ports := []string{"80", fmt.Sprintf("443%s", d.CertsString)}
```

Comment:
That looks good. My apologies for the gofmt mess. It would be best to run gofmt once from the build container and check that in. That should be standard no matter what's on our machines.